### DISCUSSION TITLE
being able to use external for images

### DISCUSSION THREAD

**Discussion by josephisticated** [maximmax42/Discord-CustomRP#99] [Ideas]
being able to use external sources for images instead of uploading them to discord and choosing the key on the app, discord now allows third party sources to be shown on rich presence and some other rich presence software already has the feature, it would have been nice to see it here too

**Comment by maximmax42** [comment]
Turns out you can already do that since 1.14, but I never knew about it until now. I'll add the info about that to the setup manual.
